### DISCUSSION TITLE
Lesson 11: Can't get image into 'img' folder

### DISCUSSION THREAD

**Discussion by PhoenixSatoshi** [Cyfrin/foundry-full-course-cu#678] [Q&A] [answered]
I have been trying to get the dog image into my 'img' folder. Nothing seems to do the trick. I have tried drag and dropping, pasting in the image and have even downloaded the image paste extension. Does anyone have any idea what I'm doing wrong?

**Comment by 0xusmanf** [comment, marked as answer]
Try copy-pasting manually through file explorer.

**Reply by PhoenixSatoshi** [reply]
That worked, thank you.
